apps/api/src/app/integrations/integrations.controller.ts

```diff
@@ -8,12 +18,20 @@ import { CreateIntegrationBodyDto } from './dto/create-integration.dto';
 import { CreateIntegrationCommand } from './usecases/create-integration/create-integration.command';
 import { GetIntegration } from './usecases/get-integration/get-integration.usecase';
 import { GetIntegrationCommand } from './usecases/get-integration/get-integration.command';
+import { Roles } from '../auth/framework/roles.decorator';
+import { UpdateIntegrationBodyDto } from './dto/update-integration.dto';
+import { UpdateIntegration } from './usecases/update-integration/update-integration.usecase';
+import { UpdateIntegrationCommand } from './usecases/update-integration/update-integration.command';
 
 @Controller('/integrations')
 @UseInterceptors(ClassSerializerInterceptor)
 @UseGuards(JwtAuthGuard)
 export class IntegrationsController {
-  constructor(private getIntegrationUsecase: GetIntegration, private createIntegrationUsecase: CreateIntegration) {}
+  constructor(
+    private getIntegrationUsecase: GetIntegration,
```

Comment:
I thinks it's better to be `GetIntegrations` instead of integration, as it implies one integration will return